src/tasks/python/PythonExtensionHelper.ts

```diff
@@ -34,8 +35,15 @@ export namespace PythonExtensionHelper {
     export async function ensureDebuggerReady(prelaunchTask: vscode.Task, debuggerSemaphorePath: string, containerName: string, cliDockerClient: CliDockerClient): Promise<void> {
         // tslint:disable-next-line:promise-must-complete
         return new Promise((resolve, reject) => {
+            const dockerTaskListener = dockerTaskEndEventListener.event((taskEvent: DockerTaskEvent) => {
+                if (!taskEvent.success) {
+                    cleanupListeners();
+                    reject(localize('vscode-docker.tasks.pythonExt.failedToAttach', 'Failed to attach the debugger, please see the terminal output for more details.'));
```

Comment:
Does this end up as output to the task window or as a toast? If toast, I'd suggest `context.errorHandling.suppressReportIssue = true;` or we'll get tons of mistaken bug reports.

Reply:
It would end up as a toast, but unfortunately due to the convoluted nature of Python's debugging logic, to suppress the button here means we'd have to suppress up the chain here "[resolveDebugConfiguration](https://github.com/microsoft/vscode-docker/blob/ab6fdaf7c32dfdae25dcf94308b5babd65578f51/src/debugging/DockerDebugConfigurationProvider.ts#L51)", since that's where this will be called from. Is that something we'd be okay with?

Reply:
Can we pass `IActionContext` down? If we reject here it should throw it all the way up. If we suppress report issue only here then nothing else is affected.

Reply:
Turns out it is passed down in the `DockerDebugContext `object, so I've set that in Python's `resolveDebugConfiguration`.